src/main/dialogs/quickSetStrip/qssDialog.js

```diff
@@ -345,3 +410,48 @@ gpii.app.qss.handleBlur = function (that, tray, closeQssOnBlur) {
         that.hide();
     }
 };
+
+/**
+ * Initialisation for the QSS being an App Bar.
+ * @param {Component} appBar The gpii.windows.appBar instance.
+ */
+gpii.app.qss.appBarInit = function (appBar) {
+
+    if (!appBar.getPrimaryDisplay) {
+        appBar.getPrimaryDisplay = electron.screen.getPrimaryDisplay;
+    }
+
+    // Override so that the vertical window positioning calculations are relative to the taskbar top (or screen bottom),
+    // rather than the work area, because this dialog will be outside of the work area.
+    electron.screen.getPrimaryDisplay = function () {
```

Comment:
Pretty dangerous approach - do we need to patch electron's own use of this function or just our own? If the latter, we should just create a shim in our own namespace.

Reply:
done.
I don't think electron uses it - I created a shim, and it still works.